src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -90,6 +90,9 @@ public static void MapPlaceholderColor(IEditorHandler handler, IEditor editor)
 		public static void MapIsTextPredictionEnabled(IEditorHandler handler, IEditor editor) =>
 			handler.PlatformView?.UpdateIsTextPredictionEnabled(editor);
 
+		public static void MapIsSpellCheckEnabled(IEditorHandler handler, IEditor editor) =>
```

Comment:
@jfversluis Docs added on this commit! https://github.com/dotnet/maui/pull/15459/commits/c6defcc0b83a455bd9aac2d7f2170851fbee9bad 
I'm hoping that documenting the Standard implementation is sufficient to have it reflect on MSLearn and other places, though its hard to tell since I couldn't find a documented mapper in the Handlers folder 🥲